lib/ship_quiet_logistics/commands/send_shipment.rb

```diff
@@ -0,0 +1,35 @@
+module ShipQuietLogistics
+  module Commands
+    class SendShipment
+      def self.call(shipment)
+        new(shipment).call
+      end
+
+      def initialize(shipment)
+        @shipment = shipment
+        @config = ShipQuietLogistics.configuration
+      end
+
+      def call
+        Api.send_document('ShipmentOrder',
+                          shipment,
+                          config.outgoing_bucket,
+                          config.outgoing_queue,
+                          configs)
+
+        shipment.update(pushed: true)
```

Comment:
Where does this attribute get surfaced in the backoffice, or is that a decision left up to the host app?